_posts/2019-04-14-ferryman-post.md

```diff
@@ -0,0 +1,53 @@
+
+
+## Unity
+
+The built in collaboration tool was surprisingly great! We are accustomed [Git](https://github.com/SylverStudios) at work and for personal projects, but we tried the Unity built in collaboration tool and ended up sticking with it. There were a couple bumps along the way, but we highly recommend trying it out.
+
+We made a heavily menu based game that would have been easy for us in React, or Elm, but in Unity we were blind. After a good amount of fighting with C#, we did end up with a couple working pages, but I expect there is a much easier way to build UI. C# isn't to blame for that, but we unanimously enjoy the languages that we write usually, but the support of Unity makes C# worth it.
+
+
+## Design
+
+Art takes so much longer than you expect it to! We worked on the project for about 15 hours, and it's crazy how few assets came out of that. In retrospect, there is also so much more to the aesthetic of your game than just the few images of game pieces. The designer's role can encompass the entire feel of the game. With [The Ferryman]({{ site.baseurl }}{% post_url 2019-04-10-ferryman %}) we had a lot of text, menus, and backgrounds.
+
+Choosing a color pallet and letting Sam and RJ get at it just wasn't enough to give the in-game Bar the feeling that we wanted. Next time around, I think the design role will be a more holistic job, focused on putting less effort into more aspects of the game, instead of lots of effort into a handful of assets.
+
+
+## Timeline
+
+This part was awesome for us. Historically, we have had very soft deadlines, if any at all. The deadline was non-negotiable and it was short, we knew exactly how much effort we would be able to put in before losing our sanity. We prioritized ruthlessly, cutting out many "fun", but inessential features. At the end of Saturday morning, we had a plan for the minimal features that we could still consider a "game."
+
+Of course this didn't lead to the worlds most exciting game, but that wasn't our goal. We wanted to challenge ourself to plan well, and execute. That has been a problem in the past for us, and this was the best opportunity we've had to really test out or self-control.
+
+
+## So, what happened at the end?
+
+Sunday afternoon, wrapping up the game, trying to build a WebGL version to publish on the website and we realized we didn't read the rules very closely… **An additional rule farther down the page said, "no text".** Obviously that is a huge rule, and seeing as our game was basically all text, we couldn't submit. Despite how disqualifying our mistake was, we were on a mission to challenge ourselves, and we absolutely crushed it.
```

Comment:
A lot of the humor here comes from the tragic aspect. We could focus at least a bit on the _moment_ of tragedy, when we all looked at each other like "wtf did we just do for the past 24 hours lmao". Certainly, we can come back to "yes but it was definitely a positive experience" after

Reply:
I have an attempt incoming, but maybe could use some assistance from someone that knows more about build-up and dramatic pause - @rjdellecese